bench/index.ts

```diff
@@ -0,0 +1,81 @@
+import chalk from "chalk";
+
+import { BenchFixture } from "./fixture";
+
+async function main() {
+  const fixture = await BenchFixture.create();
+
+  console.log(chalk.bold("=== Settlement Gas Benchmarks ==="));
+  console.log(
+    chalk.gray(
```

Comment:
Also very nitpicky: this gray is hard to tell apart from the black in my terminal! But I love the use of colors.

Reply:
Is gray OK for the lines? I could just change the header colors then.